packages/edit-post/src/components/layout/index.js

```diff
@@ -405,6 +291,25 @@ function Layout( { initialPost } ) {
 					}
 				/>
 			) }
+			<PostEditorMoreMenu />
+			<BackButton initialPost={ initialPost } />
+			<EditorCanvas
+				className={ className }
+				styles={ styles }
+				forceIsDirty={ showMetaBoxes }
+				contentRef={ paddingAppenderRef }
+				disableIframe={ ! shouldIframe }
+				// We should auto-focus the canvas (title) on load.
+				// eslint-disable-next-line jsx-a11y/no-autofocus
+				autoFocus={ ! isWelcomeGuideVisible }
+			>
+				{ ! isDistractionFree && showMetaBoxes && (
+					<div className="edit-post-layout__metaboxes">
+						<MetaBoxes location="normal" />
+						<MetaBoxes location="advanced" />
+					</div>
+				) }
+			</EditorCanvas>
```

Comment:
I wish we didn't need all these props (forceIsDirty, children, disableIframe, autoFocus...) but I think there's no way around them for now (mostly to inject metaboxes UI...)